.vscode/launch.json

```diff
@@ -133,8 +133,10 @@
       // "runtimeExecutable": "~/.nvm/versions/node/v8.11.2/bin/node",
       "cwd": "${workspaceFolder}/packages/integration",
       "runtimeArgs": [
+        "-r",
```

Comment:
Is ts-node really needed to run integration package ?
Integration specs are all coded with js. ts-node is needed for code loaded from specs but not by integration itself ?